index.js

```diff
@@ -191,7 +191,8 @@ async function underPressure (fastify, opts) {
     return {
       eventLoopDelay,
       rssBytes,
-      heapUsed
+      heapUsed,
+      eventLoopUtilizationVal
```

Comment:
Can you add a test for this change?

Reply:
Yeah I will do

Reply:
I have added the test. Please take a look

Reply:
Why does it have `Val` at the end? Could you remove that? Thanks.

Reply:
It's the internal variable used to calculate the event loop utilization metric. Do you want to change all the occurrences in the source files or else I can change the object here as `{eventLoopUtilizationMetric: eventLoopUtilizationVal}`. Since we had already released a version, removing all occurrences will be a breaking change, won't it?  Thoughts? @mcollina

Reply:
@mcollina

Reply:
> Since we had already released a version, removing all occurrences will be a breaking change, won't it? 

Are you sure that `eventLoopUtilizationVal` was exposed when the release has been done?
(is doesn't seem to me)

Reply:
Oh my bad. I got confused. Will update the PR accordingly. Sorry for the confusion :)

Reply:
Updated. Please take a look @Eomm